pom.xml

```diff
@@ -66,140 +66,144 @@
         <sonar.exclusions>**/autogen/**/*</sonar.exclusions>
         
         <!-- 3rd party library versions -->
-        <guava.version>32.1.2-jre</guava.version>
-        <checker-qual.version>3.39.0</checker-qual.version>
-        <error_prone_annotations.version>2.22.0</error_prone_annotations.version>
+        <guava.version>33.4.0-jre</guava.version>
+        <checker-qual.version>3.48.4</checker-qual.version>
+        <error_prone_annotations.version>2.36.0</error_prone_annotations.version>
         <j2objc-annotations.version>1.3</j2objc-annotations.version>
         
-        <commons-lang3.version>3.15.0</commons-lang3.version>
-        <commons-codec.version>1.16.0</commons-codec.version>
+        <commons-lang3.version>3.17.0</commons-lang3.version>
+        <commons-codec.version>1.17.2</commons-codec.version>
         <commons-math3.version>3.6.1</commons-math3.version>
-        <caffeine.version>2.9.3</caffeine.version>
-        <transmittable-thread-local.version>2.14.2</transmittable-thread-local.version>
-        <java-util.version>2.4.0</java-util.version>
+        <commons-compress.version>1.27.1</commons-compress.version>
+        <commons-configuration2.version>2.11.0</commons-configuration2.version>
+        <commons-text.version>1.13.0</commons-text.version>
+        <caffeine.version>3.2.0</caffeine.version>
```

Comment:
- Isn't the last version of caffeine that supports running on `JDK 8` 2.9.3? Why is this change necessary?

Reply:
This brings up a new topic, why do you need to try to update hadoop? Only hiveserver2 jdbc driver requires hadoop, and hiveserver2 jdbc driver does not support hadoop 3.4.0 at all.